App/StackExchange.DataExplorer/packages.config

```diff
@@ -17,12 +17,18 @@
   <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net451" />
   <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net451" />
   <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Configuration" version="2.0.0" targetFramework="net48" />
+  <package id="Microsoft.Extensions.Configuration.Abstractions" version="2.0.0" targetFramework="net48" />
+  <package id="Microsoft.Extensions.Configuration.Binder" version="2.0.0" targetFramework="net48" />
+  <package id="Microsoft.Extensions.Primitives" version="2.0.0" targetFramework="net48" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net451" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net40" />
   <package id="MiniProfiler" version="3.2.0.157" targetFramework="net451" />
   <package id="MiniProfiler.MVC4" version="3.0.11" targetFramework="net451" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net451" />
-  <package id="StackExchange.Exceptional" version="1.0.122" targetFramework="net451" />
+  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net48" />
+  <package id="StackExchange.Exceptional" version="2.2.32" targetFramework="net48" />
+  <package id="StackExchange.Exceptional.Shared" version="2.2.32" targetFramework="net48" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.4.0" targetFramework="net48" />

```

Comment:
Same here - recommend moving to `<PackageReference>` and eliminating most of this, then you'll just need for example `<PackageReference Include="StackExchange.Exceptional" Version="2.2.32" />` in the project file and nothing here removing everything in this file - that'll transitively flow to the test project making it simpler as well.

Not sure if you're familiar but if not happy to PR to this PR to demo what it looks like!

Reply:
@NickCraver  I thought `<PackageReference>` wasn't supported for .NET Full Framework ASP.NET projects -- has that changed? Looks like the built-in migration tool refuses to migrate the ASP.NET project as well.  I can certainly do this for the test project, but I don't think it works for the web project.  Please enlighten me if I'm wrong.

Reply:
(ignore my previous comment about it not being supported in ASP.NET full framework projects... looks like that's been fixed)